frontend/src/main/components/Commons/CommonsList.js

```diff
@@ -28,15 +29,15 @@ const CommonsList = (props) => {
                 <Card.Subtitle>
                     <Container>
                         <Row>
-                            <Col data-testid="commonsList-subtitle-id" sx={4}>ID#</Col>
+                            {/* <Col data-testid="commonsList-subtitle-id" sx={4}>ID#</Col> */}
                             <Col data-testid="commonsList-subtitle-name" sx={4}>Common's Name</Col>
                             <Col sm={4}></Col>
                         </Row>
                     </Container>
                 </Card.Subtitle>
                 {
                     props.commonList.map(
-                        (c) => (<CommonsCard key={c.id} commons={c} buttonText={props.buttonText} buttonLink={props.buttonLink} />)
+                        (c) => (<CommonsCard key={c.id} commons={c} buttonText={props.buttonText} buttonLink={props.buttonLink} color = {colors[c.id]}/>)
```

Comment:
What is your solution when the is no longer in the range of your array here?

This is not going to work.

There is a solution that will, but I'm going to leave it up to you to figure that out.    Hint: mod operator.

Reply:
The solution I had in mind was 

 props.commonList.map(
                        (c) => (<CommonsCard key={c.id} commons={c} buttonText={props.buttonText} buttonLink={props.buttonLink} color = {colors[c.id % colors.length ]}/>)